llvm/test/CodeGen/AArch64/extractvector-of-load.mir

```diff
@@ -0,0 +1,91 @@
+# RUN: llc -run-pass=aarch64-prelegalizer-combiner %s -o - | FileCheck %s
```

Comment:
might want -verify-machineinstrs or whatever

also a lot of people use the update_mir_test_checks script for GISel testcases

Reply:
Done